python_modules/dagster/dagster/_core/workspace/autodiscovery.py

```diff
@@ -114,7 +114,7 @@ def loadable_targets_from_loaded_module(module: ModuleType) -> Sequence[Loadable
         return [LoadableTarget(LOAD_ALL_ASSETS, [*module_assets, *module_source_assets])]
 
     raise DagsterInvariantViolationError(
-        "No Definitions, function decorated with @definitions, RepositoryDefinition, Job, Pipeline, Graph, or AssetsDefinition found in "
+        "No Definitions, function decorated with @lazy_definitions, RepositoryDefinition, Job, Pipeline, Graph, or AssetsDefinition found in "
```

Comment:
I don't think this string should include `lazy_definitions` as it user-facing.

The first one in this file is ok because only users of lazy_definitions will see it.